eventdef/eventdef.go

```diff
@@ -57,12 +95,111 @@ func Save(e *schema.ProbeEvent) error {
 	if err := e.Validate(); err != nil {
 		return err
 	}
+	
+	t := time.Now()
+	y, m, d := t.Date()
+	idxName := fmt.Sprintf("events-%d-%02d-%02d", y, m, d)
```

Comment:
the indexName needs to be based on the event timestamp, not on when it gets written.